src/components/interactive-builder/edit-question.modal.tsx

```diff
@@ -149,8 +149,11 @@ const EditQuestionModal: React.FC<EditQuestionModalProps> = ({
     questionToEdit?.questionOptions.concept &&
     questionToEdit?.questionOptions?.concept !== selectedConcept?.uuid;
   const [addInlineDate, setAddInlineDate] = useState(false);
-  const [toggleLabelTrue, setToggleLabelTrue] = useState(questionToEdit?.questionOptions?.toggleOptions?.labelTrue);
-  const [toggleLabelFalse, setToggleLabelFalse] = useState(questionToEdit?.questionOptions?.toggleOptions?.labelFalse);
+  const [toggleLabelTrue, setToggleLabelTrue] = useState('');
+  const [toggleLabelFalse, setToggleLabelFalse] = useState('');
+  const [formMarkdown, setFormMarkdown] = useState([]);
+  const [buttonLabel, setButtonLabel] = useState('');
+  const [workspaceName, setWorkspaceName] = useState('');
```

Comment:
These variables should be initiated not with an empty string but with the relevant values from the `questionToEdit` object